src/components/containers/ContainersDetailContent.js

```diff
@@ -60,7 +60,25 @@ const ContainersDetailContent = ({containersByID, samplesByID, get, listParents}
           </Descriptions.Item>
           <Descriptions.Item label="Kind">{container.kind}</Descriptions.Item>
           <Descriptions.Item label="Comment" span={3}>{container.comment}</Descriptions.Item>
-          <Descriptions.Item label="Sample(s)" span={3}>
+          <Descriptions.Item label="Content" span={3}>
+            {(container.children && container.children.length > 0 )?
+              <div><b>Container(s)</b></div>
+              : ""}
```

Comment:
Usually the form `{condition && element}` is preferred over `{condition ? element : ""}`.